src/ela_did.h

```diff
@@ -522,11 +527,11 @@ DID_API DIDDocument *DID_Resolve(DID *did, bool force);
  * @param
  *      did                      [in] The handle of DID.
  * @return
- *      If no error occurs, return the handle to DID Document buffter. Free
+ *      If no error occurs, return the handle to DIDHistory. Free
  *      Otherwise, return NULL.
- * @ User need to free the return value and destroy every DID Document.
+ * @ User need to destory didhistory.
```

Comment:
* @return
*            when no error occurs, it returns the handle to DIDHistory instance, otherwise, it returns NULL. Notice that user need to release the handle of returned instance to destroy it's memory.